home/models.py

```diff
@@ -1,7 +1,37 @@
 from django.db import models
+from wagtail.admin.panels import MultiFieldPanel, FieldPanel
+from wagtail.fields import RichTextField
 
 from wagtail.models import Page
 
+from wagtail.models import Page
+from wagtail.fields import RichTextField
+
+# import MultiFieldPanel:
+from wagtail.admin.panels import FieldPanel, MultiFieldPanel
```

Comment:
Some of these imports are redundant

I think PyCharm can fix that automatically, try `Ctrl`+`Alt`+`Shift`+`L`, or double tapping `Shift` and searching for "Reformat Code" or similar

Reply:
Didnt work for me but I manually removed unused imports.